### PR TITLE
Peeringdb fix

### LINKED ISSUE #772: PeeringdbNetID returning null
image

The peeringDB link on ASOverview component is returning undefined because the query used to get the peeringdbNetID is returning null from the IYP.

image

### PR DESCRIPTION
**Pull request by 0xAnon0602**
Fixes #772 

For AS 55836, it now shows like this 

image

### CODE REVIEW

File: src/components/networks/as/ASOverview.vue [hunk 1]
```diff
@@ -113,7 +113,7 @@ const handleReference = (key) => {
   } else if (key === 'bgp.tools') {
     externalLink = `${references.value[key]}/${asn}`
   } else if (key === 'peeringdb.com') {
-    externalLink = `${references.value[key]}/${pdbid.value}`
+    if(pdbid.value) externalLink = `${references.value[key]}/${pdbid.value}`
```
Comment (dpgiakatos) [inline src/components/networks/as/ASOverview.vue:116]: Delete the if statement

File: src/components/networks/as/ASOverview.vue [hunk 2]
```diff
@@ -113,7 +113,7 @@ const handleReference = (key) => {
   } else if (key === 'bgp.tools') {
     externalLink = `${references.value[key]}/${asn}`
   } else if (key === 'peeringdb.com') {
```
Comment (dpgiakatos) [inline src/components/networks/as/ASOverview.vue:115]: Add here the below condition. E.g.:
```
...
} else if (key === 'peeringdb.com' && pdbid.value) {
...
```

**Review by dpgiakatos — CHANGES_REQUESTED** [review]
Please address the issues mentioned below. Thanks.

**Comment by 0xAnon0602** [comment]
@dpgiakatos 
Fixed it with recent commit )

**Review by dpgiakatos — APPROVED** [review]
Thanks

**Pull request merged**